index.html

```diff
@@ -19,16 +19,32 @@
             <div class="menu">
                 <a href="#" class="humberger"><i class="fas fa-bars"></i></a>
                 <div class="menu-content">
-                  <a href="#">Portfolio</a>
-                  <a href="#">About</a>
-                  <a href="#">Contact</a>
+                  <a href="#portfolio">Portfolio</a>
+                  <a href="#about">About</a>
+                  <a href="#contact">Contact</a>
```

Comment:
- [ ] Only clicking the portfolio menu works but others are not getting me to the appropriate section of the page, they seem to be not working, would you address that? 

https://user-images.githubusercontent.com/55918152/140830145-054b2b8c-4ecb-497f-a3a5-656486657fcf.mp4

Whereas that shouldn't be the case, whenever a user clicks on any of the menu, he/she should be taken to the appropriate section, would you implement this? That will increase your site accessibily and great User experience!